STPopup/STPopupController.m

```diff
@@ -169,7 +169,6 @@ - (void)setupObservers
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(orientationDidChange) name:UIApplicationDidChangeStatusBarOrientationNotification object:nil];
     
     // Observe keyboard
-    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(keyboardWillShow:) name:UIKeyboardWillShowNotification object:nil];
```

Comment:
@Igor-Palaguta Actually it subscribes to two different events: UIKeyboardWillShowNotification and UIKeyboardWillChangeFrameNotification with the same method: keyboardWillShow. So think it's still necessary to have both of them.